war/pom.xml

```diff
@@ -736,7 +740,7 @@ THE SOFTWARE.
                   <goal>gulp</goal>
                 </goals>
                 <configuration>
-                  <arguments>lint bundle</arguments>
+                  <arguments>jshint bundle</arguments>
```

Comment:
But both versions are compatible with this change according to the build log, right?